frontend/www/js/omegaup/components/teamsgroup/Teams.test.ts

```diff
@@ -2,9 +2,34 @@ import { mount, shallowMount } from '@vue/test-utils';
 
 import { dao, types } from '../../api_types';
 
-import teamsgroup_Teams from './Teams.vue';
+import teamsgroup_Teams, { AvailableForms } from './Teams.vue';
+import identity_Edit from '../identity/Edit.vue';
+import identity_ChangePassword from '../identity/ChangePassword.vue';
 
 describe('Members.vue', () => {
+  beforeEach(() => {
+    const div = document.createElement('div');
+    div.id = 'root';
+    document.body.appendChild(div);
+  });
+
+  afterAll(() => {
+    const rootDiv = document.getElementById('root');
+    if (rootDiv) {
+      document.removeChild(rootDiv);
+    }
+  });
+
+  const propsData = {
+    teams: [
+      {
+        username: 'omegaUp:user',
+        name: 'user',
+      },
+    ] as types.Identity[],
+    countries: [{ country_id: 'mx', name: 'Mexico' }] as dao.Countries[],
+  };
```

Comment:
```suggestion
  const propsData: {
    teams: types.Identity[];
    countries: dao.Countries[];
  } = {
    teams: [
      {
        username: 'omegaUp:user',
        name: 'user',
      },
    ],
    countries: [{ country_id: 'mx', name: 'Mexico' }],
  };
```

recuerda: hay que evitar el uso de `as` en declaraciones. en esos casos, hay que declarar el tipo después del nombre de la variable.

en general hay que evitar el uso de `as` porque TypeScript deja de checar el tipo y acepta lo que se le proporciona (casi) sin cuestionar, _excepto_ cuando se le pasa a los props directamente.

esto se va a mejorar con Vue 3 que tiene mejor soporte de TypeScript. _posiblemente_ podríamos eliminar el uso de `as` permanentemente!